src/plans/plan-interface.h

```diff
@@ -42,9 +42,15 @@ LibraryAction DigCircular (In Real X,
                            In Real GroundPos,
                            In Boolean Parallel);
 
-LibraryAction DeliverSample (In Real X,
-                             In Real Y,
-                             In Real Z);
+LibraryAction DigLinear (In Real X,
```

Comment:
Our coding guidelines say this should be named "digLinear". There are a lot of capitalized function names in here that should be lower-case.

lander-commands.h contains a corresponding `dig_linear` function. This is pretty confusing naming. Wouldn't it be better to give them more distinct names or put each of them inside a descriptive namespace? Or are these PLEXIL requirements that I don't understand?

Reply:
These particular examples follow the PLEXIL coding guidelines in our Confluence.  We can discuss if your interpretation of that section is different.